modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+
+* With this update, the default installation namespace for {gitops-title} Operator has been changed to its own namespace called `openshift-gitops-operator`. You can still choose the old default installation namespace, `openshift-operators`, through a drop-down menu available in the OperatorHub UI at installation time. You can also enable cluster monitoring on the new namespace by selecting the check box, which makes the Operator's performance metrics accessible within the {product-title} web console. link:https://issues.redhat.com/browse/GITOPS-3073[GITOPS-3073]
++
+[NOTE]
+====
+The {gitops-title} Operator's metrics are only available when the Operator is installed in the default namespace, `openshift-gitops-operator`.
+====
+
+* Before this update, there was no option for scaling the application controller replicas dynamically. Now, you can dynamically scale the number of application controllers based on the number of clusters managed by each application controller. link:https://issues.redhat.com/browse/GITOPS-3287[GITOPS-3287]
+
+* Before this update, there was no option for choosing an algorithm for distributing the destination clusters equally across the different application controller shards. Now, you can set the sharding algorithm to `round-robin` parameter, which distributes clusters equally across the different application controller shards so that the synchronization load is spread equally among the shards. link:https://issues.redhat.com/browse/GITOPS-3288[GITOPS-3288]
+
+* With this update, the {gitops-title} Operator exports custom metrics that allow you to track the performance of the Operator. The following are the exported metrics:
+** `active_argocd_instances_total` - The number of Argo CD instances currently managed across the cluster.
+** `active_argocd_instances_by_phase{phase="<_PHASE>"}` - The number of managed Argo CD instances in the given phase.
+** `active_argocd_instance_reconciliation_count{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The number of times the instance in a given namespace is reconciled.
+** `controller_runtime_reconcile_time_seconds_per_instance{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The distribution of reconciliation cycles by their duration for the instance in a given namespace.
+link:https://issues.redhat.com/browse/GITOPS-3071[GITOPS-3071]
+
+
+[id="deprecated-features-1-10-0_{context}"]
+== Deprecated and removed features
+
+* With this release, the following deprecated `sso` and `dex` fields are removed from Argo CD CR:
+** The `.spec.sso.image`, `.spec.sso.version`, `.spec.sso.resources`, and `.spec.sso.verifyTLS` fields for keycloak SSO configurations.
+** The `.spec.dex` fields, along with `DISABLE_DEX` environment variable.
```

Comment:
I am not sure why this was not addressed since the comment was marked as resolved  (same for the first item list). Here it is the same as I mentioned above:
IBM style guide: If list items comprise only sentence fragments, do not include any end punctuation.
```suggestion
** The `.spec.sso.image`, `.spec.sso.version`, `.spec.sso.resources`, and `.spec.sso.verifyTLS` fields for keycloak SSO configurations
** The `.spec.dex` fields, along with `DISABLE_DEX` environment variable
```

Reply:
@ekristova,I had addressed it, but @Srivaralakshmi advised that panctuation marks should be returned, that is why you are seeing them again

Reply:
done

Reply:
I see, thank your for the explanation @masauso-lungu !

Reply:
It was my mistake that I did not quote the changes I was requesting first time, so now I made sure to explain every change with IBM guide

Reply:
Thank you @ekristova